pkgs/by-name/el/elf2uf2-rs/package.nix

```
@@ -1,33 +1,38 @@
-{ lib, stdenv, rustPlatform, fetchCrate, pkg-config, udev, CoreFoundation, DiskArbitration, Foundation }:
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchCrate,
+  pkg-config,
+  udev,
+}:
 
 rustPlatform.buildRustPackage rec {
   pname = "elf2uf2-rs";
   version = "2.0.0";
```

```suggestion
  version = "2.1.1";
```
Everything else LGTM